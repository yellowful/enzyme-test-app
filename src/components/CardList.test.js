import React from "react";
import { shallow } from "enzyme";
import CardList from "./CardList"

it('mocking props',()=>{
    const robots = [
        {
            id:10,
            name:"jack",
            email:"jack@gmail.com"
        }
    ]
    expect(shallow(<CardList robots={robots} />)).toMatchSnapshot();
})