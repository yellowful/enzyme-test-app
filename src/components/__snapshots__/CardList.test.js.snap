// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mocking props 1`] = `
<div>
  <Card
    email="jack@gmail.com"
    id={10}
    key="0"
    name="jack"
  />
</div>
`;
